guava/src/com/google/common/math/Quantiles.java

```diff
@@ -274,7 +281,12 @@ public double compute(int... dataset) {
      *     be arbitrarily reordered by this method call
      * @return the quantile value
      */
-    public double computeInPlace(double... dataset) {
+    @SuppressWarnings({"lowerbound:assignment.type.incompatible",// Since index and (dataset.length - 1) are non-negative ints, numerator is non negative.
```

Comment:
The checker should be able to figure this out if `index` is `@NonNegative` and `dataset` is `@MinLen(1)`. Can you produce a short test case that shows that this inference fails? I'd like to have you fix it if the checker is really unable to do this (the fix should be straightforward).

Reply:
@kelloggm [Link to test case](https://docs.google.com/document/d/1N3u0fWkzxtwKuUQK2PFYmxoVltQ98HLZYfNpGXgdeKs/edit?usp=sharing)